### ISSUE TITLE
Jaws does not capture hand movement properly if moved into by tabbing backwards

### ISSUE THREAD

**Issue by KatieWoe** [phetsims/ratio-and-proportion#348]
For https://github.com/phetsims/QA/issues/601. Win 10 Chrome Jaws.

This was a bit difficult to track down, but I noticed that, sometimes, Jaws would not read out any alerts when moved using the two hands setting, and would only read how close they were to each other, but not position in range, when only one was moved. This seemed to be persistent once it showed up.

As best I can tell, this is how to reproduce this.
1. Using Jaws, tab into the hands on a page, but not past it, move them and observe proper alerts.
2. Tab past the hands to the tick mark selection boxes
3. Tab back and attempt to listen to alerts when both moving one hand, and when moving both.

https://user-images.githubusercontent.com/41024075/107159070-2b9b6500-694b-11eb-9835-b91029f84cde.MOV

https://user-images.githubusercontent.com/41024075/107159082-36ee9080-694b-11eb-8c53-4580cd606334.MOV

**Comment by zepumph** [comment]
Hmmm. I will need to get JAWS to experiment with this more. I won't get to it tonight.

**Comment by zepumph** [comment]
@jessegreenberg and I can consistently get aria-live alerts to be broken when using JAWS. The easiest to reproduce is whe the reset all button, which almost always successfully alerts on startup, but quickly stops and just says "enter" on reset all button press.

I tried adding the `alert` role to our paragraphs, and it worked to speak every alert very well (fixing the current problem), but I don't think this will be acceptable, because it causes all alerts to interrupt, event interrupting the aria-valuetext. It also prefaces every alert with "Alert: . . . ", actually saying "alert" at the beginning of each alert.

I'll keep appending things I try here.

**Comment by zepumph** [comment]
I was unfortunately able to reproduce this issue in https://phet.colorado.edu/sims/html/gravity-force-lab-basics/latest/gravity-force-lab-basics_en.html. I wonder what has happened with JAWS.

This patch didn't fix anything:
```diff
Index: js/AriaHerald.js
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/js/AriaHerald.js b/js/AriaHerald.js
--- a/js/AriaHerald.js	(revision a146f5ba46cf4974ffb98b0e3cb175b416b52d34)
+++ b/js/AriaHerald.js	(date 1613088387908)
@@ -167,7 +167,7 @@
         // see https://github.com/phetsims/scenery-phet/issues/490
         liveElement.hidden = true;
       }, 200 );
-    }, 0 );
+    }, 10 );
   }
 }
 
```

**Comment by zepumph** [comment]
@jessegreenberg I'm pretty stumped. Do you have more ideas for me to pursue?

**Comment by zepumph** [comment]
Wow! This pretty much fixed the problem for me on JAWS, though it looks like it was added for iOS. . . So I'm not sure how to proceed here.

```
Index: js/AriaHerald.js
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/js/AriaHerald.js b/js/AriaHerald.js
--- a/js/AriaHerald.js	(revision a146f5ba46cf4974ffb98b0e3cb175b416b52d34)
+++ b/js/AriaHerald.js	(date 1613093166416)
@@ -151,7 +151,7 @@
     liveElement.textContent = '';
 
     // element must be visible for alerts to be spoken
-    liveElement.hidden = false;
+    // liveElement.hidden = false;
 
     // must be done asynchronously from setting hidden above or else the screen reader
     // will fail to read the content
@@ -165,7 +165,7 @@
 
         // Using `hidden` rather than clearing textContent works better on mobile VO,
         // see https://github.com/phetsims/scenery-phet/issues/490
-        liveElement.hidden = true;
+        // liveElement.hidden = true;
       }, 200 );
     }, 0 );
   }

```

**Comment by zepumph** [comment]
I found and reopened https://github.com/phetsims/scenery-phet/issues/490. I'll try to pick things up over there.

**Comment by zepumph** [comment]
After a bit more testing, I should just need to cherry-pick https://github.com/phetsims/utterance-queue/commit/8104d517b9720d5111e9e2e453f0010c9fa188af.

**Comment by zepumph** [comment]
After some more testing on NVDA and JAWS on firefox and chrome, I feel ready to do this. I'll cherry pick.

**Comment by KatieWoe** [comment]
This seems fixed after a quick check during rc.2. Will reopen if something comes up during full Jaws testing.
